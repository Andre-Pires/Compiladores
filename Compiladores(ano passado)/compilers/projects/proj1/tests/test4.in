/* test4, mistake in comment * /

int f(int x){
  int y; int z;
  char c;
  if(x < 1)then return 1;else{y=x - 1;z=f(y);return x * z;}
}
