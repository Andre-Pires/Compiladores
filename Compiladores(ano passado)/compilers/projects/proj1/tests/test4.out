#Total Number of Tokens Matched was 0
